pkg/config/config_test.go

```diff
@@ -58,6 +58,15 @@ resources:
     warning:
       below: 300M
       above: 4G
+images:
+  tagNotSpecified: error
+  pullPolicyNotAlways: warning
+  whitelist:
+    error:
+      - gcr.io/*
```

Comment:
Thanks for adding these @kimschles! It looks as though you're setting them in the config, and the config parses correctly, but we're not actually validating the settings are parsed correctly in `testParsedConfig()`. This means, the YAML/JSON will be read, but it's not actually checking the values against anything serialized into Go structs.

I am going to write up an example here (below) on how one might achieve that, to assure we're loading what we expect to load from the yaml & json.